GrandCentralBoard/GoogleAPIDataProvider.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  GoogleAPIDataProvider.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 06.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Alamofire
+
+enum GoogleAPIDataError: ErrorType {
```

Comment:
See how the rest of the project is formatted. It's a suggestion though. Up to you.

Btw. Apple made this once:

![1cea928](https://cloud.githubusercontent.com/assets/3382607/14320132/35fadd4a-fc14-11e5-9ff2-96c7d3176dd6.jpg)